apps/app/src/components/AppContainer.tsx

```diff
@@ -30,6 +31,8 @@ export const AppContainer = (props: AppProps) => {
   return (
     <Flowbite theme={{ dark: true }}>
       <QueryClientProvider client={queryClient}>
+        <Toaster expand={false} />
```

Comment:
In the past this was moved elsewhere due to intl errors (the toaster couldn't access translations), but things may have changed - if this fixes the issue you described on your commit message and you don't see any intl errors on a full app build (pnpm build - pnpm start), then great!